containers/.env.template

```diff
@@ -0,0 +1,32 @@
+# This template file indicates the environment variables that are not explicitly set by
```

Comment:
A brief environment template folder that only works with containers -- this does not work in the project root. If we like the idea of .env.template files I could create something for the project root also.